tests/Requester.py

```diff
@@ -263,6 +263,39 @@ def testBaseUrlPrefixRedirection(self):
             "Following Github server redirection from /api/v3/repos/PyGithub/PyGithub to /repos/PyGithub/PyGithub"
         )
 
+    def testMakeAbsoluteUrl(self):
+        class TestAuth(github.Auth.AppAuth):
+            pass
```

Comment:
Do we really need this sub-class, can't we use `AppAuth` here directly?

Reply:
Right, I updated it to use `github.Auth.AppAuth` directly.